src/test/java/org/datadog/jmxfetch/TestApp.java

```diff
@@ -494,7 +491,7 @@ public void testServiceCheckCounter() throws Exception {
         // Let's put a service check in the pipeline (we cannot call doIteration()
         // here unfortunately because it would call reportStatus which will flush
         // the count to the jmx_status.yaml file and reset the counter.
-        repo.sendServiceCheck("jmx", Status.STATUS_OK, "This is a test", "jmx_test_instance", null);
+        repo.sendServiceCheck("jmx", Status.STATUS_OK, "This is a test", null);
 
         // Let's check that the counter has been updated
         assertEquals(1, repo.getServiceCheckCount("jmx"));
```

Comment:
Hmm actually all of our tests rely on the Attach API and thus don't use a `host` in the configuration, so we can't cleanly test the `jmx_server` tag with our current test setup...
We could change our test setup to use JMX Remote instead but that would need some work.